test/Sentry.Tests/Internals/Http/CachingTransportTests.cs

```diff
@@ -257,5 +258,69 @@ public async Task DoesNotDeleteCacheIfConnectionWithIssue()
             Assert.Equal(exception, receivedException);
             Assert.True(Directory.EnumerateFiles(cacheDirectory.Path, "*", SearchOption.AllDirectories).Any());
         }
+
+        [Fact(Timeout = 7000)]
+        public async Task IsolatesCacheDirectories()
```

Comment:
Is it possible to rephrase this test name? from what I understand, it tests different processes generating envelopes and it expects to have all those envelopes to be stored on their given isolated process directory?